src/js/engagement_view/src/components/engagementView/sidebar/tables/lensTable/lensTable.tsx

```diff
@@ -15,6 +15,8 @@ import { PaginationState } from "types/CustomTypes";
 
 export const lensTable = (
     state: PaginationState,
+    selectedIdState: number,
+    setSelectedIdState: (selectedIdState: number) => void,
```

Comment:
preface for the rest of this PR: I am not a react expert

I guess I'm confused why there's additional state outside the variable named `state`
could we rename that to `paginationState`?

Reply:
Sorry, this is unclear - this state refers to the selected lens, not selected pagination. We have two separate states here, one for pagination, one for current selected lens. Will rename the variables to be more clear.